### ISSUE TITLE
ANSI::CYAN etc are throwing an error on the query_proxy.rb file.

### ISSUE THREAD

**Issue by skunkworker** [neo4jrb/neo4j#906]
When trying to generate a query ex `post.comments` I get an error on the query_proxy.rb object.
This is the only place that I can find that `ANSI` is being used, is there a dependency missing?
```
2.2.2 :001 > Post.first.comments
 CYPHER 4ms MATCH (n:`Post`) RETURN n ORDER BY n.uuid LIMIT {limit_1} | {:limit_1=>1}
NameError: uninitialized constant Neo4j::ActiveNode::Query::QueryProxy::ANSI
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/neo4j-5.0.13/lib/neo4j/active_node/query/query_proxy.rb:54:in `inspect'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/neo4j-5.0.13/lib/neo4j/active_node/has_n.rb:27:in `inspect'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/railties-4.2.3/lib/rails/commands/console.rb:110:in `start'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/railties-4.2.3/lib/rails/commands/console.rb:9:in `start'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/railties-4.2.3/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/railties-4.2.3/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/john/.rvm/gems/ruby-2.2.2/gems/railties-4.2.3/lib/rails/commands.rb:17:in `<top (required)>'
	from /Users/john/dev/neo4j/bin/rails:8:in `<top (required)>'
	from /Users/john/.rvm/rubies/ruby-2.2.2/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/john/.rvm/rubies/ruby-2.2.2/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

**Comment by cheerfulstoic** [comment]
Sorry for the delay!

Also sorry for this bug!  This comes from a problem with cherry-picking some code from master into the 5.0.x branch which depended on code which remained in `neo4j-core` master.  I've just cherry-picked the code which is needed into the 5.0.x branch of `neo4j-core` and released `5.0.10`, so could you `bundle update neo4j-core` and see if that fixes it?

Thanks

**Comment by geezyx** [comment]
Her there, just FYI I had the same issue when querying a "has_many" association of a node.  Updating to 5.0.10 fixed the problem for me, from 5.0.9 where I had the issue.

Thank you!

**Comment by cheerfulstoic** [comment]
Awesome, thanks!  I'll close for now.  FYI there's also a `5.0.15` (it's been a busy week) and we just released `5.1.0.rc.1` if you want to try that out
